### ISSUE TITLE
[Question]: Fine-tuning a Spanish model : is it worth it ?

### ISSUE THREAD

**Issue by ZoeNoel** [flairNLP/flair#3219]
### Question

Hello,

We are beginners with Flair and we use it for NER tasks. We have already used the Spanish model ("ner-spanish-large") on our 19th c. corpus and we reach an accuracy of 94% and a F1-Score of 92%.
Do you think that fine-tuning a model will improve these results or it will be worst as we have already good results?

We have already tried several things for fine-tuning: #[HunFlair Tutorial 2: Training NER models](https://github.com/flairNLP/flair/blob/master/resources/docs/HUNFLAIR_TUTORIAL_2_TRAINING.md), #[Tutorial 9: Training your own Flair Embeddings](https://github.com/flairNLP/flair/blob/master/resources/docs/TUTORIAL_9_TRAINING_LM_EMBEDDINGS.md) and  issue [#53](https://github.com/flairNLP/flair/issues/53), issue #[1178](https://github.com/flairNLP/flair/issues/1178), issue #2758 . However, we cannot get any results. Here is the error message:

![Capture d’écran 2023-04-25 155937](https://user-images.githubusercontent.com/117484256/234308422-6d743691-8284-4fa5-ac68-699ab3d31dce.png)

We think there might be a problem with the construction of our corpus. How do we have to prepare our corpus for fine-tune with Flair? We tried to cut our corpus in three parts: test, train and valid, following what other tutorials did, but it did not work. Our files are in CoNLL 2002 format as we correct them with Inception.

Thanks for your help !

**Comment by helpmefindaname** [comment]
Hi @ZoeNoel 
I think there are 2 concepts mixed up:
A `TextCorpus` is used for the Language Model Pretraining, if I understand your case correctly, you don't want to train a Language Model but finetune a NER-Model.
For the NER-Model you can look at [this tutorial](https://flairnlp.github.io/docs/tutorial-training/how-to-train-sequence-tagger#training-a-named-entity-recognition-ner-model-with-transformers) and for the loading of the corpus you can look at [this one](https://flairnlp.github.io/docs/tutorial-training/how-to-load-custom-dataset)

**Comment by ZoeNoel** [comment]
Hi @helpmefindaname,

Thank you for your answer and your advice, it works perfectly !

We fine-tuned a model that we found on Hugging-face (https://huggingface.co/mrm8488/bert-spanish-cased-finetuned-ner) and it is fine, but we weren't able to use the model "ner-spanish-large" of Flair that we used at the beginning of our experiments. When we tried, we obtained an error saying that a config.json file is missing. Is it possible to fine tune this model?

Thanks again !

**Comment by helpmefindaname** [comment]
Can you share the code that lead you to that error?

**Comment by ZoeNoel** [comment]
The code is available on the project's github, [here](https://github.com/DesenrollandoElCordel/pliegos-ner/blob/main/flair_fine_tuning.ipynb). This code works with the "bert-spanish-cased-finetuned-ner" model but, when we replaced this model with "flair/ner-spanish-large", we got the error saying that the config.json file is missing.
Thank for your help!

**Comment by helpmefindaname** [comment]
So you are trying to load the `flair/ner-spansih-large` *SequenceTagger* as *TransformerWordEmbedding*?

Instead you should load the SequenceTagger and just extract the embedding: `embeddings=SequenceTagger.load("flair/ner-spanish-large).embeddings`.

**Comment by ZoeNoel** [comment]
Thank you, it works !
